python/covid_web/covid_web/country_dashboard.py

```diff
@@ -170,16 +171,16 @@ def _create_buttons(
         return buttons
 
     def _create_single_country_callbacks(self, app: dash.Dash) -> None:
-        # TODO: Predictions will be generated statically in the future.
         graphs_by_country = {
             country_short_name: CountryGraph(
                 self.report_by_short_name[country_short_name],
-                get_fitted_predictions(
-                    self.report_by_short_name[country_short_name],
+                # TODO(mszabados): Make it possible to select only automatic predictions.
+                self.prediction_db.select_predictions(
+                    country=country_short_name,
                     last_data_dates=self.report_by_short_name[country_short_name].dates[-1:],
```

Comment:
The range here was really short, so that the prediction computation doesn't take ages. We can extend it after your change.
```suggestion
                    last_data_dates=self.report_by_short_name[country_short_name].dates[-28:],
```

Reply:
We can have all with the slider.